ivy/functional/frontends/paddle/math.py

```diff
@@ -110,6 +110,14 @@ def atanh(x, name=None):
     return ivy.atanh(x)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle"
+)
+@to_ivy_arrays_and_back
+def broadcast_shape(x):
```

Comment:
Hey! not sure if I missed anything, but according to the docs there should be 2 parameters: https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/broadcast_shape_en.html#broadcast-shape
Thanks!

Reply:
Ok, sure. I would fix that